### ISSUE TITLE
Add Carthage support?

### ISSUE THREAD

**Issue by emartinson** [rundfunk47/stinsen#87]
Could you please add Carthage support apart from SPM and Cocoapods?

**Comment by rundfunk47** [comment]
What is needed for Carthage support? If I remember correctly, it should work out of the box, by just specifying the repository URL and target?

**Comment by emartinson** [comment]
No, to support Carthage publisher should provide XCode Project with shared schema which can product XCFramework as build result.

Without this carthage will fail with `Dependency "stinsen" has no shared framework schemes`
